src/Microsoft.PowerShell.Commands.Utility/commands/utility/MatchString.cs

```diff
@@ -315,8 +315,8 @@ public string ToEmphasizedString(string directory)
         /// <returns>The matched line with matched text inverted.</returns>
         private string EmphasizeLine()
         {
-            string invertColorsVT100 = VTUtility.GetEscapeSequence(VTUtility.VT.Inverse);
-            string resetVT100 = VTUtility.GetEscapeSequence(VTUtility.VT.Reset);
+            string invertColorsVT100 = PSStyle.Instance.Reverse;
```

Comment:
Why do we need the `Instance`? We can convert PSStyle to static.

Reply:
Because the variable `$PSStyle` has to point to an object, which is the singleton object `PSStyle.Instance`.

Reply:
At least the members can be static if Instance is only needed for $PSStyle?

Reply:
Static members cannot be accessed via `$PSStyle.<xxx>` syntax. Instead, you have to use `[xxxx]::<member-name>`. We want the syntax `$PSStyle.<xxx>` works.